app/models/settings/general.rb

```diff
@@ -35,8 +35,6 @@ class General < Base
             default: proc { URL.local_image("icon.png") },
             validates: { url: true }
 
-    setting :logo_svg, type: :string
```

Comment:
This is minor but if we didn't do this in a previous PR, we may want to also remove the setting from the DB. This is the line we'd need for the DUS:

```ruby
Settings::General.destroy_by(var: :logo_svg)
```

Reply:
Got it; done!